rollup.config.js

```diff
@@ -16,7 +16,11 @@ const plugins = {
 	typescript: typescript({
 		tsconfig: 'tsconfig.code.json'
 	}),
-	commonjs: commonjs(),
+	commonjs: commonjs({
+		namedExports: {
+			'timsort': ['sort']
+		}
```

Comment:
Timsort exports sort but Rollup fails to detect this so I had to set it here manually.